### PR TITLE
Adding base of oop salad homework.

### PR DESCRIPTION
**Pull request by CarlosGuadamuz0**

### CODE REVIEW

File: src/main/java/oop_chef_homework/chef/Chef.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+import oop_chef_homework.salad.Salad;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Chef {
+    private Salad salad;
+
+    public Chef() {
+        salad = null;
```
Comment (BohdanKalep) [inline src/main/java/oop_chef_homework/chef/Chef.java:14]: No sense to make it null 
https://www.codejava.net/java-core/the-java-language/java-default-initialization-of-instance-variables-and-initialization-blocks

File: src/main/java/oop_chef_homework/chef/Chef.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    }
+
+    /**
+     * Displays a menu and handles user input for various Chef actions.
+     */
+    public void menu() {
```
Comment (BohdanKalep) [inline src/main/java/oop_chef_homework/chef/Chef.java:20]: Update method name in accroding to code conventions and divide this method into smaller methods

File: src/main/java/oop_chef_homework/ingredients/Vegetable.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
+/**
+ * @author Carlos Guadamuz
+ */
+package oop_chef_homework.ingredients;
+
+
+public class Vegetable implements VegetableInterface {
+    private final String NAME;
```
Comment (BohdanKalep) [inline src/main/java/oop_chef_homework/ingredients/Vegetable.java:8]: in is not necessary to make it final

File: src/main/java/oop_chef_homework/ingredients/Vegetable.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        NAME = this.getClass().getSimpleName();
+        this.CALORIES = CALORIES;
+    }
+
+    /**
+     * Gets the calorie content of the vegetable.
+     *
+     * @return The calorie content.
+     */
+    @Override
```
Comment (BohdanKalep) [inline src/main/java/oop_chef_homework/ingredients/Vegetable.java:26]: no need to have override annotation

File: src/main/java/oop_chef_homework/ingredients/Vegetable.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+        return this.CALORIES;
+    }
+
+    /**
+     * Gets the inheritance hierarchy of the vegetable's classes.
+     *
+     * @return A string representing the inheritance hierarchy.
+     */
+    @Override
+    public String getInheritance() {
```
Comment (BohdanKalep) [inline src/main/java/oop_chef_homework/ingredients/Vegetable.java:37]: no need to have override annotation

File: src/main/java/oop_chef_homework/ingredients/VegetableManager.java [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+import oop_chef_homework.ingredients.vegetables.stem.Asparagus;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class VegetableManager {
+    private static final List<Vegetable> VEGETABLES = new ArrayList<>();
+
+    // Static initializer block to populate the list with vegetable instances
+    static {
```
Comment (BohdanKalep) [inline src/main/java/oop_chef_homework/ingredients/VegetableManager.java:22]: no need to use static block. Bettor to use a simple method

File: src/main/java/oop_chef_homework/main/ChefRunner.java [hunk 7]
```diff
@@ -0,0 +1,13 @@
+/**
+ * @author Carlos Guadamuz
+ */
+package oop_chef_homework.main;
+
+import oop_chef_homework.chef.Chef;
+
+public class ChefRunner {
+    public static void main(String[] args) {
+        Chef chef=new Chef();
```
Comment (BohdanKalep) [inline src/main/java/oop_chef_homework/main/ChefRunner.java:10]: add spaces

**Comment by CarlosGuadamuz0** [comment]
I think I fixed everything, thanks for the feedback

**Review by BohdanKalep — APPROVED** [review]

**Pull request merged**